src/runtime/opengl.cpp

```diff
@@ -14,110 +14,112 @@
 
 // List of all OpenGL functions used by the runtime. The list is used to
 // declare and initialize the dispatch table in OpenGLState below.
-#define USED_GL_FUNCTIONS                                               \
```

Comment:
Is all this reformatting just clang-format? (it makes it hard to review the meaningful changes... bulk reformatting like this is better done on its own to avoid that issue)